composer.json

```diff
@@ -17,10 +17,10 @@
     ],
     "require": {
         "php": "^7.2",
+        "laminas/laminas-cache": "^2.7",
+        "laminas/laminas-stdlib": "^2.2 || ^3.0",
         "league/flysystem": "^1.0.41",
```

Comment:
@basz should we change to latest release (1.0.64)?

Reply:
any particular reason why we should? I don't see why...

Reply:
I'm not composer specialist, but I'm wondering if it's possible to require the minimum version (1.0.41) but if nothing is specificed in the project the composer will install the latest stable version in 1.0.* (at the moment, 1.0.64).

Make sense?

Reply:
The caret (^) mean any from 1.0.41 and below 2.0.0. So we are good.